### PR TITLE
Update authentication.md

### PR DESCRIPTION
**Pull request by mertenar**
couple spelling/grammar items, feel free to review yourself if you feel differently

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/get-ready/authentication.md [hunk 1]
```diff
@@ -25,22 +25,22 @@ This topic explains the things you need to check in order to ensure that apps an
...
-### Single Sign-On by using UPN and passwords
+### Single Sign-On by using UPN and password
 
 In most organizations, your users will be able to use SSO to authenticate by UPN and password on Microsoft Managed Desktop Devices. However, to make sure this will work, you should double-check the following:
 
 - Confirm that Azure Active Directory (AAD) Connect is set up and uses an on-premises Active Directory server running Windows Server 2008 R2 or later.
 - Confirm that AAD Connect is running a supported version and is set to sync these three attributes with Azure AD: 
     - DNS domain name of the on-premises Active Directory (where the end-users are located)
-    - NetBIOS of yor on-premises Active Directory (where th end-users are located)
+    - NetBIOS of yor on-premises Active Directory (where the end-users are located)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/authentication.md]: NetBIOS of **your** on-premises Active Directory (where the end-users are located)

Please do correction as indicated in **bold** (your).

**Comment by mypil** [comment]
@mertenar - Can you possibly merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Review by mertenar — COMMENTED** [review]
corrected typo identified by JohanFreelancer9

**Pull request merged**